packages/components/src/MultiSelect/subcomponents/MultiSelectOptions/MultiSelectOptions.tsx

```diff
@@ -46,7 +47,7 @@ export const MultiSelectOptions = ({
         Options available: {options.length}
       </VisuallyHidden>
       {options.length === 0 ? (
-        <span>No options available</span>
+        <Text variant="body">No options available</Text>
```

Comment:
Since `Text` outputs a `p` by default, should it instead be a `span`?

Reply:
We just had a placeholder. Doesn't hurt to be a `p` :)

Reply:
Rightio, just making sure 😄